Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -121,8 +121,16 @@ def audit
     end
 
     return if problem_count.zero?
+    errors_summary = "#{Formatter.pluralize(problem_count, "problem")} in #{Formatter.pluralize(formula_count, "formula")}"
+    create_pr_comment("name-of-repo", "Pull Request Number", errors_summary) if new_formula
+    ofail errors_summary unless new_formula
+  end
 
-    ofail "#{Formatter.pluralize(problem_count, "problem")} in #{Formatter.pluralize(formula_count, "formula")}"
+  def create_pr_comment(repo, number, body)
```

Comment:
I reckon this should live in `github.rb`. Also it should ideally avoid creating duplicate comments; perhaps it's worth accessing the existing comments on a PR and only creating a comment if the string-identical comment doesn't already exist?